bin/critterget.py

```diff
@@ -554,6 +554,50 @@ def getDailyCrashes(appId,appName):
     return
 
 
+def getTrends(appId,appName):
+    """Get the number of daily app crashes for all versions of an app."""
```

Comment:
We should really give lists of args, raises, and returns.  Please add these to this and other functions you're introducing in this PR.